### ISSUE TITLE
Running db patching precheck operation for GI on DB system doesn't invoke prechecks

### ISSUE THREAD

**Issue by rishimahajan** [oracle/oci-cli#223]
Trying to run patch precheck operation for GI patch on a  DB system using cli. The call doesn't report any failure but prechecks  are not invoked on the database. Don't see any job submitted to db agent on the server. 
Tested it for few servers, the issue is only with db system (GI patching). The prechecks for database is working fine. The precheck as well as apply operation for GI works fine from OCI console. 

The issue is not patch specific.

```
[opc@tower dbpatching]$ oci --version
2.6.11
[opc@tower dbpatching]$ 

[opc@tower dbpatching]$ oci db patch list by-db-system --db-system-id "ocid1.dbsystem.oc1.iad.abuwcljtbdjwmubo6v5r3qzni2shjyr2vgpl3vzllwkd3buj4gjfzhiuxbgq" --auth instance_principal
{
  "data": [
    {
      "available-actions": [
        "APPLY", 
        "PRECHECK"
      ], 
      "description": "Oct 2019 18c Db System patch", 
      "id": "ocid1.dbpatch.oc1.iad.abuwcljt34bbre33fwaejgaeqzadrgobvf5wer75fq6npqqphsqn4daqbm7q", 
      "last-action": null, 
      "lifecycle-details": null, 
      "lifecycle-state": null, 
      "time-released": "2019-10-14T01:00:00+00:00", 
      "version": "18.8.0.0.191015"
    }
  ], 
  "opc-next-page": "AAAAAAAAAAJleUpyYVdRaU9pSTNPVElpTENKbGJtTWlPaUpCTWpVMlIwTk5JaXdpWVd4bklqb2laR2x5SW4wLi5HLVdSSEcyVFVQOHhIMTB3LkJEZGZYX0J1NlhUMER2MV83Y2lNMjdzdVJvVDVwQUJSbHhNb1B6ME10TGZxdk1XNjV0bm14Zy1KTFJCM21oeTVabWVpbDM5TGZUNE11SnRobkxfb0VuTExkZUdUX1IyMkw1ODRRNUo2N3JNQnhXbHFFNUNzQlBKMDRBdmdXVXNXbm1PeVJYcWxpdHhwWHZ1NEU4VEZtUUJWWFh4eXBBTVRMeGZSZlVJaHdNYUFDMHZVc2E1UlNZMEpIWGJpY0R2cGpldVhqT3ZtV28xT1pJZVo2RU1palBSWC1mTEtOQ3VuRzktNk9hWEVKdlptTUw1VFQtQzd2UlhHSWtlRnA3TWdVb0RIMmstcHJtdUVXX1FpNVc4WldtdmdxdzJlYjd2VndPekZIbXJMT0l0VWlMYlRPNTdGRGJ0QnZDSHFlbWpSY25kQm0wM2Y3ZzhqQ18tN0g3NWwya2hDb2t1YzdSQ0k1ZzNydXBqZHlWVjIzS0V5S2doUk9mcHJWUlltRERIOEREV3JIc2MtWi12a0p4Q2NpZEd1ZVVhb3ItSUpqNnJfUDVFUk9WOWZUN21yOGNfN2tnLmhzVG9xSE9RZTFybE02VVY4SzhJREE="
}

[opc@tower dbpatching]$ oci db system  patch --db-system-id "ocid1.dbsystem.oc1.iad.abuwcljtbdjwmubo6v5r3qzni2shjyr2vgpl3vzllwkd3buj4gjfzhiuxbgq" --patch-action 'PRECHECK' --patch-id "ocid1.dbpatch.oc1.iad.abuwcljt34bbre33fwaejgaeqzadrgobvf5wer75fq6npqqphsqn4daqbm7q" --auth instance_principal
{
  "data": {
    "availability-domain": "aBLb:US-ASHBURN-AD-1", 
    "backup-network-nsg-ids": null, 
    "backup-subnet-id": null, 
    "cluster-name": null, 
    "compartment-id": "ocid1.compartment.oc1..aaaaaaaaf6tcpns5nwjmdwl7chw4bgjwl7x7xskvipcchplw67fbiy3blazq", 
    "cpu-core-count": 1, 
    "data-storage-percentage": 80, 
    "data-storage-size-in-gbs": 1024, 
    "database-edition": "ENTERPRISE_EDITION_HIGH_PERFORMANCE", 
    "db-system-options": {
      "storage-management": "ASM"
    }, 
    "defined-tags": {}, 
    "disk-redundancy": "HIGH", 
    "display-name": "DB1", 
    "domain": "projsubnet.projvcn.oraclevcn.com", 
    "fault-domains": [
      "FAULT-DOMAIN-3"
    ], 
    "freeform-tags": {
      "costcenter": "100", 
      "environment": "dev"
    }, 
    "hostname": "db1", 
    "id": "ocid1.dbsystem.oc1.iad.abuwcljtbdjwmubo6v5r3qzni2shjyr2vgpl3vzllwkd3buj4gjfzhiuxbgq", 
    "iorm-config-cache": null, 
    "last-patch-history-entry-id": null, 
    "license-model": "LICENSE_INCLUDED", 
    "lifecycle-details": null, 
    "lifecycle-state": "AVAILABLE", 
    "listener-port": 1521, 
    "node-count": 1, 
    "nsg-ids": null, 
    "reco-storage-size-in-gb": 512, 
    "scan-dns-record-id": "ocid1.vcndnsrecord.oc1.iad.abuwcljtl2xnrz34cu2sdjxzb7udsaxve7iwhdzn22jcskqe7s3qzhkoefoq", 
    "scan-ip-ids": null, 
    "shape": "VM.Standard2.1", 
    "sparse-diskgroup": null, 
    "ssh-public-keys": [
      "ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEA54CQthF1ocKcl0uLpZcPIx4802si84Zbo6aSsbvT1xEUEbhEkKXFxl8kBFYrF812neTzFIq8XDiNYlyiWuAvQEm085zLkV/FdghUnJhT9BzPtsmf4f2yU/pV9w6L1BbnUC9msB00IGbjyrNE6ZQsQ+HD2kQJfENTVqs+6vhjsDMPKh2SD4Y4uHJGqxXX42gLU7FSZQiV+/gxO+fkNFYQLpBgyBQimozNO9IfXhtKb7RM+Vg3sm2MXtPRilRVwPtatBRPX0EFD6ui5T0Igo6jKJbJf8Sg+t0R4M5ccrYT8HhqaoeMmzK/nk2IJI9ndwmFWQYdtOILd+fP49bdhboMuQ== rsa-key-20170802"
    ], 
    "subnet-id": "ocid1.subnet.oc1.iad.aaaaaaaa5l3tmbqbholtdsupbckiue6x5an2sggoqplioejje2mvrrbufcfq", 
    "time-created": "2019-11-20T03:03:41.382000+00:00", 
    "time-zone": "UTC", 
    "version": "18.6.0.0.190416", 
    "vip-ids": null
  }
}
[opc@tower dbpatching]$ 

[root@db1 ~]# dbcli list-jobs

ID                                       Description                                                                 Created                             Status    
---------------------------------------- --------------------------------------------------------------------------- ----------------------------------- ----------
9ea5cd6f-2205-4d6f-8424-71b3273475ea     Authentication key update for DCS_ADMIN                                     November 20, 2019 3:20:14 AM UTC    Success   
2bd2a4c9-e67d-445f-8ed5-a2f9f571c514     Provisioning service creation                                               November 20, 2019 3:22:04 AM UTC    Success   
1fa52cb0-e4b9-4d22-abcb-1d294533b856     SSH keys update                                                             November 20, 2019 4:17:14 AM UTC    Success   
90eadd05-02f9-4c09-ae87-c6ea6b6740eb     SSH key delete                                                              November 20, 2019 4:18:58 AM UTC    Success   

[root@db1 ~]# 

```

**Comment by paul-hummel-oracle** [comment]
Hi @rishimahajan , 
I will label this for our Database team to take a look.

I wonder if the command happens asynchronously. Also, does adding "--debug" reveal any more details that might be helpful.

**Comment by rishimahajan** [comment]
Nothing in debug
```
[opc@anstwr ~]$ oci db system  patch --db-system-id "ocid1.dbsystem.oc1.iad.abuwcljtbdjwmubo6v5r3qzni2shjyr2vgpl3vzllwkd3buj4gjfzhiuxbgq" --patch-action 'PRECHECK' --patch-id "ocid1.dbpatch.oc1.iad.abuwcljt34bbre33fwaejgaeqzadrgobvf5wer75fq6npqqphsqn4daqbm7q" --auth instance_principal --debug
WARNING: Python 2 support is ending on December 31, 2019. Future versions of OCI CLI after January 2020, will not be compatible with Python 2. To avoid interruption at that time, please install OCI CLI in a compatible Python 3 environment.
DEBUG:oci_cli.cli_util: 2019-11-26 02:35:53.846321: time elapsed calling to_dict from render: 0.0010359287262
DEBUG:oci_cli.cli_util: 2019-11-26 02:35:53.847215: total memory usage before printing: 121.3KiB (124240)
{
  "data": {
    "availability-domain": "aBLb:US-ASHBURN-AD-1", 
    "backup-network-nsg-ids": null, 
    "backup-subnet-id": null, 
    "cluster-name": null, 
    "compartment-id": "ocid1.compartment.oc1..aaaaaaaaf6tcpns5nwjmdwl7chw4bgjwl7x7xskvipcchplw67fbiy3blazq", 
    "cpu-core-count": 1, 
    "data-storage-percentage": 80, 
    "data-storage-size-in-gbs": 1024, 
    "database-edition": "ENTERPRISE_EDITION_HIGH_PERFORMANCE", 
    "db-system-options": {
      "storage-management": "ASM"
    }, 
    "defined-tags": {}, 
    "disk-redundancy": "HIGH", 
    "display-name": "DB1", 
    "domain": "projsubnet.projvcn.oraclevcn.com", 
    "fault-domains": [
      "FAULT-DOMAIN-3"
    ], 
    "freeform-tags": {
      "costcenter": "100", 
      "environment": "dev"
    }, 
    "hostname": "db1", 
    "id": "ocid1.dbsystem.oc1.iad.abuwcljtbdjwmubo6v5r3qzni2shjyr2vgpl3vzllwkd3buj4gjfzhiuxbgq", 
    "iorm-config-cache": null, 
    "last-patch-history-entry-id": null, 
    "license-model": "LICENSE_INCLUDED", 
    "lifecycle-details": null, 
    "lifecycle-state": "AVAILABLE", 
    "listener-port": 1521, 
    "node-count": 1, 
    "nsg-ids": null, 
    "reco-storage-size-in-gb": 512, 
    "scan-dns-record-id": "ocid1.vcndnsrecord.oc1.iad.abuwcljtl2xnrz34cu2sdjxzb7udsaxve7iwhdzn22jcskqe7s3qzhkoefoq", 
    "scan-ip-ids": null, 
    "shape": "VM.Standard2.1", 
    "sparse-diskgroup": null, 
    "ssh-public-keys": [
      "ssh-rsa AAAAB3NzaC1yc2EAAAABJQAAAQEA54CQthF1ocKcl0uLpZcPIx4802si84Zbo6aSsbvT1xEUEbhEkKXFxl8kBFYrF812neTzFIq8XDiNYlyiWuAvQEm085zLkV/FdghUnJhT9BzPtsmf4f2yU/pV9w6L1BbnUC9msB00IGbjyrNE6ZQsQ+HD2kQJfENTVqs+6vhjsDMPKh2SD4Y4uHJGqxXX42gLU7FSZQiV+/gxO+fkNFYQLpBgyBQimozNO9IfXhtKb7RM+Vg3sm2MXtPRilRVwPtatBRPX0EFD6ui5T0Igo6jKJbJf8Sg+t0R4M5ccrYT8HhqaoeMmzK/nk2IJI9ndwmFWQYdtOILd+fP49bdhboMuQ== rsa-key-20170802"
    ], 
    "subnet-id": "ocid1.subnet.oc1.iad.aaaaaaaa5l3tmbqbholtdsupbckiue6x5an2sggoqplioejje2mvrrbufcfq", 
    "time-created": "2019-11-20T03:03:41.382000+00:00", 
    "time-zone": "UTC", 
    "version": "18.6.0.0.190416", 
    "vip-ids": null
  }
}
DEBUG:oci_cli.cli_util: 2019-11-26 02:35:53.847661: Time elapsed printing response data: 0.000297069549561
DEBUG:oci_cli.cli_util: 2019-11-26 02:35:53.847776: total memory usage after printing: 121.3KiB (124240)
```

**Comment by vish1** [comment]
@rishimahajan This issue has been fixed as part of release 2.9.5.
